### ISSUE TITLE
Do cntk universal learner support l2_reguralization_weight?

### ISSUE THREAD

**Issue by vidyatshetty** [Microsoft/CNTK#3152]
Hi,

@KeDengMS I am writing my own userLearner and my model is overfitted and I would like to improve the performance on the testing set. On testing on the training data I get  98% accuracy and I get 70% accuracy on the testing data. Is there any way I can add reguralization to the universal learner or any other way to improve the generalization error?
the below code doesn't work,
`learner = C.universal(MyPruneSgd, p.parameters,l2_regularization_weight=0.001)`
Any suggestion would help.

Thanks

**Comment by KeDengMS** [comment]
Universal learner basically let user define their computation graph with gradients and models. L2 regularization is just a weight decay and can be expressed in the user defined graph.

**Comment by vidyatshetty** [comment]
@KeDengMS  Thank you for writing. Is it possible to apply dropout while retraining the model?
I have a trained model saved. I  am trying load the model and make few changes to the loaded model and retraining the model. my question is, Is there any way I can apply dropout to the layers of the loaded model while retraining?
Any comments would help.
Thanks

**Comment by KeDengMS** [comment]
You may clone and modify the model to add drop out layers.
